core/server/data/migrations/versions/3.22/02-settings-key-renames.js

```diff
@@ -0,0 +1,81 @@
+const logging = require('../../../../../shared/logging');
+
+const renameMapping = [{
+    down: 'default_locale',
+    up: 'lang'
```

Comment:
Maybe it's just how I read the code but would `old` and `new` make this easier to understand?

Reply:
The reasoning behind this naming was to match up our `up`/`down` migration phases. Another one I went with initially was `from` / `to`. If it's very unreadable, happy to rename to any of these :+1:

Reply:
Agree from/to would be preferable